### PR TITLE
SHARK-193: reducing Shark's logging verbosity

### PR DESCRIPTION
**Pull request by AndreSchumacher**
See https://spark-project.atlassian.net/browse/SHARK-193

### CODE REVIEW

**Comment by AndreSchumacher** [comment]
Reynold, would be great if you could check whether I was too aggressive. There are still a few logInfo's left in a few places.

**Comment by rxin** [comment]
Jenkins, test this please.

File: src/main/scala/shark/execution/MapJoinOperator.scala [hunk 1]
```diff
@@ -141,9 +141,9 @@ class MapJoinOperator extends CommonJoinOperator[MapJoinDesc, HiveMapJoinOperato
       if(storageLevel == StorageLevel.NONE)
         rddForHash.unpersist()
 
-      logInfo("wrappedRows size:" + wrappedRows.size)
+      logDebug("wrappedRows size:" + wrappedRows.size)
       val collectTime = System.currentTimeMillis() - startCollect
-      logInfo("HashTable collect took " + collectTime + " ms")
+      logDebug("HashTable collect took " + collectTime + " ms")
```
Comment (rxin) [inline src/main/scala/shark/execution/MapJoinOperator.scala]: Let's keep this one for now.

File: src/main/scala/shark/execution/GroupByPreShuffleOperator.scala [hunk 2]
```diff
@@ -144,11 +144,11 @@ class GroupByPreShuffleOperator extends UnaryOperator[HiveGroupByOperator] {
       if (numRowsInput == numRowsCompareHashAggr) {
         if (numRowsHashTbl > numRowsInput * minReductionHashAggr) {
           useHashAggr = false
-          logInfo("Mapside hash aggregation disabled")
+          logDebug("Mapside hash aggregation disabled")
```
Comment (rxin) [inline src/main/scala/shark/execution/GroupByPreShuffleOperator.scala]: Let's keep this one for now

File: src/main/scala/shark/execution/GroupByPreShuffleOperator.scala [hunk 3]
```diff
@@ -144,11 +144,11 @@ class GroupByPreShuffleOperator extends UnaryOperator[HiveGroupByOperator] {
...
         } else {
-          logInfo("Mapside hash aggregation enabled")
+          logDebug("Mapside hash aggregation enabled")
```
Comment (rxin) [inline src/main/scala/shark/execution/GroupByPreShuffleOperator.scala]: Let's keep this one for now

File: src/main/scala/shark/execution/GroupByPreShuffleOperator.scala [hunk 4]
```diff
@@ -144,11 +144,11 @@ class GroupByPreShuffleOperator extends UnaryOperator[HiveGroupByOperator] {
...
         }
-        logInfo("#hash table="+numRowsHashTbl+" #rows="+
+        logDebug("#hash table="+numRowsHashTbl+" #rows="+
```
Comment (rxin) [inline src/main/scala/shark/execution/GroupByPreShuffleOperator.scala]: Let's keep this one for now

File: src/main/scala/shark/execution/ScriptOperator.scala [hunk 5]
```diff
@@ -181,10 +181,10 @@ class ScriptOperator extends UnaryOperator[HiveScriptOperator] {
     }
 
     val wrappedCmdArgs = addWrapper(cmdArgs).toSeq
-    logInfo("Executing " + wrappedCmdArgs)
-    logInfo("tablename=" + hconf.get(HiveConf.ConfVars.HIVETABLENAME.varname))
-    logInfo("partname=" + hconf.get(HiveConf.ConfVars.HIVEPARTITIONNAME.varname))
-    logInfo("alias=" + alias)
+    logDebug("Executing " + wrappedCmdArgs)
```
Comment (rxin) [inline src/main/scala/shark/execution/ScriptOperator.scala]: Let's keep these 4

File: src/main/scala/shark/execution/TableScanOperator.scala [hunk 6]
```diff
@@ -178,7 +178,7 @@ class TableScanOperator extends TopOperator[HiveTableScanOperator] with HiveTopO
 
         def prunePartitionFunc(index: Int): Boolean = {
           if (printPruneDebug) {
-            logInfo("\nPartition " + index + "\n" + indexToStats(index))
+            logDebug("\nPartition " + index + "\n" + indexToStats(index))
```
Comment (rxin) [inline src/main/scala/shark/execution/TableScanOperator.scala]: Let's keep this one for now

File: src/main/scala/shark/execution/TableScanOperator.scala [hunk 7]
```diff
@@ -192,7 +192,7 @@ class TableScanOperator extends TopOperator[HiveTableScanOperator] with HiveTopO
         // Do the pruning.
         val prunedRdd = PartitionPruningRDD.create(rdd, prunePartitionFunc)
         val timeTaken = System.currentTimeMillis - startTime
-        logInfo("Map pruning %d partitions into %s partitions took %d ms".format(
+        logDebug("Map pruning %d partitions into %s partitions took %d ms".format(
```
Comment (rxin) [inline src/main/scala/shark/execution/TableScanOperator.scala]: Let's keep this one for now

File: src/main/scala/shark/memstore2/ColumnarSerDe.scala [hunk 8]
```diff
@@ -64,7 +64,7 @@ class ColumnarSerDe extends SerDe with LogHelper {
       val rowSize = ColumnarSerDe.getFieldSize(objectInspector).toLong
       estimatedNumRows = (partitionSize / rowSize).toInt
       shouldCompress = SharkConfVars.getBoolVar(conf, SharkConfVars.COLUMNAR_COMPRESSION)
-      logInfo("Initializing column serde " +
+      logDebug("Initializing column serde " +
```
Comment (rxin) [inline src/main/scala/shark/memstore2/ColumnarSerDe.scala]: Let's keep this one for now

**Comment by rxin** [comment]
Thanks. Looks good in general. I commented on a few that we should still keep at info level.

**Comment by massie** [comment]
Sorry for the noise. Testing. Jenkins, test this please.

**Comment by rxin** [comment]
Thanks, Andre!

**Pull request merged**